synapse/storage/databases/main/event_push_actions.py

```diff
@@ -539,12 +553,11 @@ async def remove_push_actions_from_staging(self, event_id: str) -> None:
         """
 
         try:
-            res = await self.db_pool.simple_delete(
```

Comment:
This returns an `int`, but declares it returns `None`. The return value is never used.